docs/src/main/asciidoc/spanner.adoc

```diff
@@ -1343,6 +1343,22 @@ class SpringSecurityAuditorAware implements AuditorAware<User> {
 
 You can also set a custom provider for properties annotated `@LastModifiedDate` by providing a bean for `DateTimeProvider` and providing the bean name to `@EnableSpannerAuditing(dateTimeProviderRef = "customDateTimeProviderBean")`.
 
+=== Multi-Instance Usage
+
+Your application can be configured to use multiple Cloud Spanner instances or databases by providing a custom bean for `DatabaseUtilityProvider<DatabaseId>`.
+The default bean uses the instance ID, database name, and project ID options you configured in `application.properties`.
+
+[source,java]
+----
+    @Bean
+    public DatabaseUtilityProvider<DatabaseId> databaseIdProvider() {
```

Comment:
I thought we discussed and agreed on using something like `DatabaseClientProvider`. It's more flexible in a sense that it allows the app to connect to Spanner instances even in different projects.
You can always implement a `DatabaseClientProvider` that works with `DatabaseId`.

Reply:
@meltsufin , so there is a provider bean for DatabaseClient exposed. However the way DatabaseClients receive their instance/database/project id is via the DatabaseId class. furthermore there is a DatabaseAdminClient that also uses the DatabaseId 
so it made sense to also make a provider bean available for DatabaseId
(emphasizing that DatabaseClient provider bean is also exposed)

Reply:
Updated the refdoc to emphasize that a bean for Supplier of DatabaseClient is also available and an option to set. 


Speaking of Suppliers, should we go back and remove CredentialProvider and ProjectIDProvider and just use Suppler of Credentials and Supplier of ProjectID ?